cmd/generate.go

```diff
@@ -119,8 +119,14 @@ function load_cockroach() {
   roachprod run "$1" "rm -f ./cockroach"
   if [ -z "$cockroach_binary" ]
   then
-    roachprod stage "$1" cockroach
+    cockroach_version=$(curl -s -i https://edge-binaries.cockroachdb.com/cockroach/cockroach.linux-gnu-amd64.LATEST |grep location|awk -F"/" '{print $NF}')
+    echo "WARN: staging a stable cockroach binary from master with hash: 5ac733bb4927020bc1c52da24b2591742fde8e1f"
+    roachprod stage "$1" cockroach 5ac733bb4927020bc1c52da24b2591742fde8e1f
```

Comment:
this really hard codes things. Any way to resolve sha?  Or perhaps if sha specified as cockroach binary, download it?

Reply:
This is only for the initial run, right? For the final run, we'll reference an actual release?

Reply:
Have some discussion with dev-inf, this seems a more involved change which needs more discussion. The SHA version should be the best stable master build we should use, let me revisit this on how to surface the version definition in a better less hard-coded way latter when I have more band-width, opened [](https://cockroachlabs.atlassian.net/browse/CR-87) to track.